samples/webhook/src/main/kotlin/me/ivmg/webhook/CertificateUtils.kt

```diff
@@ -0,0 +1,33 @@
+package me.ivmg.webhook
+
+import java.io.File
+import java.io.FileInputStream
+import java.security.KeyStore
+
+object CertificateUtils {
+    val privateKeyPassword = "YOUR_PRIVATE_KEY_PASSWORD".toCharArray()
```

Comment:
I'm not sure if this "toCharArray" is expected; what's the motivation behind?
maybe this password could be constants and the conversion to char array done lazily when needed

Reply:
The motivation is just that the places where we use this password need a `CharArray`. And as we're currently using it in three different places, instead of having a constant `String` and transforming it to `CharArray` every time we need to use it I prefer transforming it to `CharArray` just one time and using it everywhere.